scripts/packages/BUILD

```diff
@@ -266,6 +270,42 @@ pkg_tar(
 )
 
 ################################################################################
+# Heron python packaging
+################################################################################
+genrule(
+    name = "generate-pyheron-package",
+    srcs = [
+        "pyheron/requirements.txt", 
+        "pyheron/setup.py",
+    ] + generated_release_files,
+    outs = generated_pyheron_egg_file,
+    stamp = 1,
+    tools = [
+        "//heron/pyheron/src/python:pyheron-package"
+    ],
+    cmd = "\n".join([
+        'export OUTPUT_DIR=$$(pwd)/$(@D)',
+        'export TMP_DIR=$$(mktemp -d -t pyheron.XXXXX)',
+        'echo $$TMP_DIR',
+        'export PYHERON_DIR=$$TMP_DIR/pyheron',
+        'export PYHERON_UNZIP=$$TMP_DIR/unzipped',
+        'mkdir -p $$TMP_DIR $$PYHERON_DIR',
+        'unzip -qd $$PYHERON_UNZIP $(location //heron/pyheron/src/python:pyheron-package)',
+        'find $$PYHERON_UNZIP -name "*.pyc" -exec rm {} \;',
+        'mv $$PYHERON_UNZIP/heron/pyheron/src/python $$PYHERON_DIR/pyheron',
+        'rm -rf $$PYHERON_UNZIP/heron/pyheron',
+        'mv $$PYHERON_UNZIP/heron $$PYHERON_DIR/heron',
+        'cp $(SRCS) $$PYHERON_DIR',
+        'echo "PyHeron toplevel directory: $$PYHERON_DIR"',
+        'cd $$PYHERON_DIR',
+        '/usr/bin/env python2.7 setup.py bdist_egg',
+        'ls -l $$PYHERON_DIR/dist',
+        'mv $$PYHERON_DIR/dist/pyheron-*-py2.7.egg $$OUTPUT_DIR/pyheron.egg',
```

Comment:
It seems that the name of the egg file should follow the original naming convention (`pyheron-{version number}-py2.7.egg` instead of just `pyheron.egg`), so that it can be recognized as a valid package.